apps/explorer/lib/explorer/chain.ex

```diff
@@ -4624,6 +4625,18 @@ defmodule Explorer.Chain do
     |> handle_page(paging_options)
   end
 
+  defp handle_random_access_paging_options(query, empty_options, _is_et) when empty_options in [nil, [], %{}],
+    do: limit(query, ^(@default_page_size + 1))
```

Comment:
this private function appears to be a duplicate of https://github.com/HorizenLabs/eon-explorer/pull/12/files#diff-3fce0f082a5cb927aa738ef8f7d582824df91d08957971d36a0873efacd88816R4616-R4617 but only with an extra parameter `_is_et` which appears to be unused. can this function be removed since it's already declared or is `_is_et` internally used within this block?

Reply:
elixir has a feature called pattern matching, whereby depending on what parameters the method is called with the implementation which matches those particular parameters is the one that is used. the underscore in _is_et let's the compiler know that the var won't be used in the body of the method but it will be used for the pattern match